R/getAbundanceFeature.R

```diff
@@ -0,0 +1,78 @@
+#' Returns abundance values for user specified \dQuote{FeatureID}
+#' in all \dQuote{SampleID}
+#'
+#' Basic extractor function for extracting abundances for a user
+#' specified \dQuote{FeatureID} from assay in a
+#' \code{\link[=SummarizedExperiment-class]{SummarizedExperiment}}
+#' object.
+#'
+#' @param x A
+#'  \code{\link[=SummarizedExperiment-class]{SummarizedExperiment}} object.
+#'
+#' @param feature_id A \dQuote{FeatureID} for which user wants to extract
```

Comment:
This is renamed row name. Make sure to add this in the description. See comments below